src/components/utilities.js

```diff
@@ -63,7 +63,8 @@ export function getAllValueField(fields) {
 export function forceSlashAfterHash(path) {
   let pathname = window.location.pathname;
   if (pathname[1] && pathname.includes('orcid_token')) {
-    window.location.href = window.location.origin + `#/${path}` + window.location.search;
+    pathname = pathname.replace('orcid_token', "")
```

Comment:
If this is a general-purpose function (it appears to be named generally), it would be better to make the `orcid_token` piece a function parameter so it can be used in multiple places.